securedrop/journalist_templates/base.html

```diff
@@ -33,7 +33,7 @@
     <div class="container">
       {% block header %}
       <header>
-        <a href="{{ url_for('main.index') }}" class="no-bottom-border"><img src="{{ g.logo }}" class="logo small"
+        <a aria-label="{{ gettext('Return to Main View') }}" href="{{ url_for('main.index') }}" class="no-bottom-border"><img src="{{ g.logo }}" class="logo small"
```

Comment:
I have the same reservation about the "main view" concept here as in the Source Interface.  How about this, per <https://docs.securedrop.org/en/stable/journalist.html#interacting-with-sources>:

```suggestion
        <a aria-label="{{ gettext('Return to All Sources') }}" href="{{ url_for('main.index') }}" class="no-bottom-border"><img src="{{ g.logo }}" class="logo small"
```